test/specs/helpers/buildURL.spec.js

```diff
@@ -34,10 +34,10 @@ describe('helpers::buildURL', function () {
     })).toEqual('/foo?foo[]=bar&foo[]=baz');
   });
 
-  it('should support special char params', function () {
+  it('should support special char params on keys', function () {
```

Comment:
docstring should be `support special char params on keys and values`? sort of minor but would be nice